modules/core/arrow-core-data/src/main/kotlin/arrow/core/SortedMapK.kt

```diff
@@ -48,6 +48,12 @@ data class SortedMapK<A : Comparable<A>, B>(private val map: SortedMap<A, B>) :
     }.value()
   }
 
+  fun <G, C> traverseIndexed(GA: Applicative<G>, f: (Int, B) -> Kind<G, C>): Kind<G, SortedMapK<A, C>> = GA.run {
```

Comment:
I would not use `run` scope for the whole function because we've seen the Android compiler choke on it. Instead, default to using it in the most localized spots you can, i.e. `GA.just` in the always and the `run` inside `later` to get `lazyAp`